### PR TITLE
1566 enable GitHub actions

### PR DESCRIPTION
**Pull request by murny**
#1566 

All seems to work. Github Actions is now building and running our unit tests, system tests, danger and eslint/rubocop.

Had to reuse our Docker compose solution as I wasn't able to use Github Actions Services as Solr is a bit painful (chicken and egg problem....we need our code pull down as we use some of it to configure solr....but Services are created and setup before we checkout our code..)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Jupiter
 
 [![Build Status](https://travis-ci.org/ualbertalib/jupiter.svg?branch=master)](https://travis-ci.org/ualbertalib/jupiter)
```
Comment (mbarnett) [inline README.md]: Should we take travis off the page? Or do you want to pull out everything travis related in a separate PR once this is working?

Comment (murny) [reply README.md]: Yeah I wanted to run them parallel for a bit. Just to make sure Github Actions is working really well. Then we can remove Travis in another PR

Comment (mbarnett) [reply README.md]: Makes sense to me!

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Build the docker-compose stack
+        run: docker-compose up -d
+
+      - name: Set up Ruby 2.6
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.6.x
```
Comment (pgwillia) [inline .github/workflows/push.yml]: Is it possible to do this with a matrix of Ruby versions, à la travis?  It'd be nice to have it check against current (or whatever is deployed in production) and the next version of Ruby at the same time.

Comment (murny) [reply .github/workflows/push.yml]: Yep I'd assume it would just be a completely separate Job (currently everything is under a "test" job, we could just split this out for different version or just run certain jobs). But maybe for this we should just set this for the Ruby version we want to make sure it works against so maybe we should downgrade this to 2.5.x instead 🤔(Keep forgetting we on 2.5 😆)

Comment (pgwillia) [reply .github/workflows/push.yml]: It's nice to know we **could** be on 2.6.
